.github/workflows/validate_datasets.yml

```diff
@@ -71,11 +71,11 @@ jobs:
       run: |
         LOCAL_BIN=$HOME/.local/bin
         VALIDATOR=$LOCAL_BIN/bids-validator
-        mkdir -p $LOCAL_BIN
-        export PATH="$LOCAL_BIN:$PATH"
-        echo PATH="$PATH" >> $GITHUB_ENV
```

Comment:
Removing `$HOME/.local/bin` from the path seems the likely cause of the failure to find the executable.